brainscore/benchmarks/Lee2020.py

```diff
@@ -0,0 +1,181 @@
+	        title = {Topographic deep artificial neural networks reproduce the hallmarks of the primate inferior temporal cortex face processing network},
+	        elocation-id = {2020.07.09.185116},
+	        year = {2020},
+	        doi = {10.1101/2020.07.09.185116},
+	        publisher = {Cold Spring Harbor Laboratory},
+	        abstract = {A salient characteristic of monkey inferior temporal (IT) cortex is the IT face processing network. Its hallmarks include: {\textquotedblleft}face neurons{\textquotedblright} that respond more to faces than non-face objects, strong spatial clustering of those neurons in foci at each IT anatomical level ({\textquotedblleft}face patches{\textquotedblright}), and the preferential interconnection of those foci. While some deep artificial neural networks (ANNs) are good predictors of IT neuronal responses, including face neurons, they do not explain those face network hallmarks. Here we ask if they might be explained with a simple, metabolically motivated addition to current ANN ventral stream models. Specifically, we designed and successfully trained topographic deep ANNs (TDANNs) to solve real-world visual recognition tasks (as in prior work), but, in addition, we also optimized each network to minimize a proxy for neuronal wiring length within its IT layers. We report that after this dual optimization, the model IT layers of TDANNs reproduce the hallmarks of the IT face network: the presence of face neurons, clusters of face neurons that quantitatively match those found in IT face patches, connectivity between those patches, and the emergence of face viewpoint invariance along the network hierarchy. We find that these phenomena emerge for a range of naturalistic experience, but not for highly unnatural training. Taken together, these results show that the IT face processing network could be a consequence of a basic hierarchical anatomy along the ventral stream, selection pressure on the visual system to accomplish general object categorization, and selection pressure to minimize axonal wiring length.Competing Interest StatementThe authors have declared no competing interest.},
+            URL = {https://www.biorxiv.org/content/early/2020/07/10/2020.07.09.185116},
+            eprint = {https://www.biorxiv.org/content/early/2020/07/10/2020.07.09.185116.full.pdf},
+            journal = {bioRxiv}}'''
+
+class DicarloLee2020ITTissueResponseCorrelation(BenchmarkBase):
+
+    def __init__(self, metric=ks_similarity, bootstrap_samples=100_000, num_sample_arrays_candidate=10, ceiler_fn=InternalConsistency):
+
+        def load_target_assembly():
+            assembly = make_static(brainscore.get_assembly('dicarlo.MajajHong2015').sel(region='IT'))
+            return assembly[:, np.in1d(assembly.presentation.image_id, list(set(stimulus_set['image_id'])))]
+
+
+        stimulus_set = Rajalingham2019()._target_assembly.stimulus_set

```

Comment:
I think this should be moved down one line and be `stimulus_set = assembly.stimulus_set`